src/components/sbb-action-group/sbb-action-group.stories.js

```diff
@@ -1,85 +1,77 @@
 import { h } from 'jsx-dom';
 import readme from './readme.md';
 
-const firstButtonTemplate = (size, alignSelf) => (
-  <sbb-button size={size} align-self={alignSelf} variant="secondary">
+const firstButtonTemplate = (alignSelf) => (
```

Comment:
maybe it's better to call it secondaryButtonTemplate and the other just ButtonTemplate